opensrp-app/src/main/java/org/smartregister/login/task/RemoteLoginTask.java

```diff
@@ -140,6 +142,11 @@ protected LoginResponse doInBackground(Void... params) {
                         }
 
                     }
+
+                    // Save the registered ANM
+                    getOpenSRPContext().allSharedPreferences().updateANMUserName(mUsername);
```

Comment:
Is this call required here (maybe it is)? 

It is also invoked at a later time once the authentication is successful, afterLoginCheck listener fires and [this line is executed](https://github.com/OpenSRP/opensrp-client-core/blob/d7a5b4b8146c363b9d43a43c5bdf862ef32d46f5/opensrp-app/src/main/java/org/smartregister/login/interactor/BaseLoginInteractor.java#L148)